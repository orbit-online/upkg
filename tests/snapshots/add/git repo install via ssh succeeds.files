[drwxr-xr-x]  .
├── [drwxr-xr-x]  bin
│   └── [lrwxrwxrwx]  acme.bin -> ../lib/upkg/.upkg/.bin/acme.bin
└── [drwxr-xr-x]  lib
    └── [drwxr-xr-x]  upkg
        ├── [drwxr-xr-x]  .upkg
        │   ├── [drwxr-xr-x]  .bin
        │   │   └── [lrwxrwxrwx]  acme.bin -> ../.packages/acme-empty@1a76bbe01f5af96acb6fe468ca5dea6878bbb6cc/bin/acme.bin
        │   ├── [drwxr-xr-x]  .packages
        │   │   └── [drwxr-xr-x]  acme-empty@1a76bbe01f5af96acb6fe468ca5dea6878bbb6cc
        │   │       ├── [drwxr-xr-x]  bin
        │   │       │   └── [-rwxr-xr-x]  acme.bin
        │   │       └── [-rw-r--r--]  upkg.json
        │   └── [lrwxrwxrwx]  acme-empty -> .packages/acme-empty@1a76bbe01f5af96acb6fe468ca5dea6878bbb6cc
        └── [-rw-r--r--]  upkg.json
