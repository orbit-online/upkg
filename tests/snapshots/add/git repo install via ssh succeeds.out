upkg: Linking 'acme.bin'
upkg: Added 'package-fixtures:$BATS_RUN_TMPDIR/package-fixtures/default/acme.git'
